src/tools/runk/libcontainer/src/cgroup.rs

```diff
@@ -29,12 +20,42 @@ fn remove_cgroup_dir(path: &Path) -> Result<()> {
             thread::sleep(delay);
         }
 
-        if !path.exists() || fs::remove_dir(path).is_ok() {
+        if cgroup.delete().is_ok() {
             return Ok(());
         }
 
         retries -= 1;
     }
 
-    return Err(anyhow!("failed to remove cgroups paths: {:?}", path));
+    return Err(anyhow!("failed to remove cgroups paths"));
```

Comment:
When we try to delete a container that is `paused` status, we can see the error message like below.
Could you modify the message for users who try to delete a container with `paused` status?
I think a simple idea is printing the error message with status or only when the status is `paused` the detailed message is printed. 

```bash
$ sudo runk run test          
$ runk pause test
$ sudo runk delete -f test
ERROR: failed to remove cgroups paths
```

Reply:
Maybe I found a better solution.  https://github.com/opencontainers/runc/pull/3217
Thaw the frozen container can ensure deleting successfully.

Reply:
@cyyzero 
Good solution! looks good me.